__snapshots__/packages/mcdoc/test-out/runtime/checker.spec.js

```diff
@@ -617,6 +653,223 @@ exports['mcdoc runtime checker typeDefinition “struct { id: string, ...struct
   }
 ]
 
+exports['mcdoc runtime checker typeDefinition “struct { id: string, data: struct { test: struct { config: double }, other: struct { baz: boolean } }[[id]] }” with value {"id":"fallback","data":{"baz":true}} 1'] = []
+
+exports['mcdoc runtime checker typeDefinition “struct { id: string, data: struct { test: struct { config: double }, other: struct { baz: boolean } }[[id]] }” with value {"id":"fallback","data":{"config":"hello"}} 1'] = [
+  {
+    "kind": "type_mismatch",
+    "node": {
+      "originalNode": "hello",
+      "inferredType": {
+        "kind": "literal",
+        "value": {
+          "kind": "string",
+          "value": "hello"
+        }
+      }
+    },
+    "expected": {
+      "kind": "double"
+    }
+  }
+]
+
+exports['mcdoc runtime checker typeDefinition “struct { id: string, data: struct { test: struct { config: double }, other: struct { baz: boolean } }[[id]] }” with value {"id":"fallback","data":{}} 1'] = [
+  {
+    "kind": "missing_key",
+    "node": {
+      "originalNode": {},
+      "inferredType": {
+        "kind": "struct",
+        "fields": []
+      }
+    },
+    "key": {
+      "kind": "literal",
+      "value": {
+        "kind": "string",
+        "value": "config"
+      }
+    }
+  }
+]
```

Comment:
This one is still wrong, not sure it works correctly on main though.

Expected would be a `some_missing_keys` error, since we aren't sure if `baz` or `config` is missing.

Reply:
It seems to be an issue caused by the error condenser, where it incorrectly sets `onlyCommonMissing` to "missing key config"

Reply:
Fixed.